src/protocols/interaction_model/StatusCodeList.h

```diff
@@ -69,5 +69,6 @@ CHIP_IM_STATUS_CODE(UnsupportedEvent      , UNSUPPORTED_EVENT       , 0xc7)
 CHIP_IM_STATUS_CODE(PathsExhausted        , PATHS_EXHAUSTED         , 0xc8)
 CHIP_IM_STATUS_CODE(TimedRequestMismatch  , TIMED_REQUEST_MISMATCH  , 0xc9)
 CHIP_IM_STATUS_CODE(FailsafeRequired      , FAILSAFE_REQUIRED       , 0xca)
+CHIP_IM_STATUS_CODE(InvalidInState        , INVALID_IN_STATE        , 0xcb)
```

Comment:
You need to add this to `src/app/util/af-enums.h` too, to EmberAfStatus.  We really need to get rid of that thing....

Reply:
This has been added!